code/modules/busy_space/organizations.dm

```diff
@@ -775,16 +1088,64 @@
 	work = "governing polity of humanity's Confederation"
 	headquarters = "Luna, Sol"
 	motto = "Nil Mortalibus Ardui Est" // Latin, because latin.  Says 'Nothing is too steep for mortals'.
-	autogenerate_destination_names = TRUE
+	legit = 95 //government
+
+	missions = list(
+		new /datum/lore/mission("SCG-T", list("transport", "passenger transport", "prisoner transport", "refugee resettlement", "classified"), ATC_TRANS),
+		new /datum/lore/mission("SCG-F", list("emergency resupply", "hazardous material transport", "strategic resupply", "classified"), ATC_FREIGHT),
+		new /datum/lore/mission("SCG-E", list("damage assesment", "data recovery", "classified"), ATC_SALVAGE),
+		new /datum/lore/mission("SCG-D", list("defense", "patrol", "military response", "joint exercise", "classified"), ATC_DEF),
+		new /datum/lore/mission/prebuilt/scientific("SCG-S"),
+		new /datum/lore/mission/prebuilt/diplomatic("SCG-D"), //local zaddat colonies, independent earth nations, internal diplomatic missions to embassies in Sol, abels rest. actual foreign diplomacy happens elsewhere
+		new /datum/lore/mission/prebuilt/medical_response("SCG-M"),
+		new /datum/lore/mission/prebuilt/defense_response("SCG-D"),
+		)
+
+	serviced = list(
+		/datum/lore/system/vir = 10,
+		/datum/lore/system/sol = 25,
+		/datum/lore/system/alpha_centauri = 20,
+		/datum/lore/system/tau_ceti = 20,
+		/datum/lore/system/kess_gendar = 20,
+		/datum/lore/system/saint_columbia = 20,
+		/datum/lore/system/jahans_post = 15,
+		/datum/lore/system/abels_rest = 15,
+		/datum/lore/system/el = 10,
+		/datum/lore/system/raphael = 10,
+		/datum/lore/system/altair = 10,
+		/datum/lore/system/zhu_que = 10,
+		/datum/lore/system/love = 5,
+		/datum/lore/system/isavaus_gamble = 3,
+		/datum/lore/system/whythe = 3, //not in scg space but im sure they can still get at it
```

Comment:
```suggestion
		/datum/lore/system/whythe = 3 //not in scg space but im sure they can still get at it
```
More reasonable, but again, should they be going to Whythe?

Reply:
i'll double check with cere just to be sure but i think that weight should be lower for 100% sure